### PR TITLE
build: grpc version>=1.46.0

### PR DESCRIPTION
**Pull request by deepankarm**
Goals:

- ...
- ...
- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/CONTRIBUTING.md#documentation-guidelines) and ask the team.

### CODE REVIEW

File: extra-requirements.txt [hunk 1]
```diff
@@ -44,8 +44,8 @@ lz4<3.1.2:                  perf, standard,devel
 uvloop:                     perf, standard,devel
 numpy:                      core
 protobuf>=3.19.1:           core
-grpcio>=1.33.1,<1.44.0:     core
-grpcio-reflection>=1.33.1,<1.44.0:  core
+grpcio>=1.44.0:             core
```
Comment (numb3r3) [inline extra-requirements.txt:47]: better to skip `v1.44.0`

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: extra-requirements.txt (hunk 1 above)
Comment (numb3r3) [reply extra-requirements.txt:47]: or remove the version bump

**Review by numb3r3 — DISMISSED** [review]
LGTM 👍

**Review by jacobowitz — APPROVED** [review]
lets wait for the tests and merge if they pass

**Pull request merged**